### PR TITLE
fix: issue-#494 about scheduler updates and bug fixed

### PR DESCRIPTION
**Pull request by badboy-huaqiao**
close issue-#494
Signed-off-by: Huaqiao Zhang <huaqiaoz@vmware.com>

If your build fails due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-ui-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [ ] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

**Comment by badboy-huaqiao** [comment]
@cloudxxx8 please review it, and do we need to push another PR to branch v2.0? it's also a bug for branch v2.0, if do, what should I do?

**Comment by badboy-huaqiao** [comment]
@bnevis-i can you review it?  @cloudxxx8 didn't reply to me.

**Comment by cloudxxx8** [comment]
@badboy-huaqiao yes, we need to fix in jakarta branch if it is a bug, because it's a LTS version

File: web/src/app/scheduler/intervalaction/edit-interval-action/edit-interval-action.component.ts [hunk 1]
```diff
@@ -187,6 +197,7 @@ export class EditIntervalActionComponent implements OnInit {
     if (!isNaN(value) && (parseFloat(value) === parseInt(value))) {
       return true
     }
+    console.log(isNaN(value))
```
Comment (bnevis-i) [inline web/src/app/scheduler/intervalaction/edit-interval-action/edit-interval-action.component.ts:200]: Should this have been removed?

**Review by bnevis-i — DISMISSED** [review]
Found a potential debugging statement that probably should have been removed.

**Comment by lenny-intel** [comment]
> @badboy-huaqiao yes, we need to fix in jakarta branch if it is a bug, because it's a LTS version

@cloudxxx8, @badboy-huaqiao , it is a High or Critical bug that doesn't have a work around that would justify fixing in Jakarta?

**Comment by badboy-huaqiao** [comment]
> > @badboy-huaqiao yes, we need to fix in jakarta branch if it is a bug, because it's a LTS version
> 
> @cloudxxx8, @badboy-huaqiao , it is a High or Critical bug that doesn't have a work around that would justify fixing in Jakarta?
> 
> Also per the Jakarta LTS, the UI is not included in the LTS https://wiki.edgexfoundry.org/pages/viewpage.action?pageId=70948690

yes, I saw it, the GUI is not included in the LTS, but we know the GUI, like EdgeX itself, is not backward compatible, which means that users must use the same version of the GUI as EdgeX, I think this bug has caused the schedule module to be unusable, it should be treated as a serious bug.

File: web/src/app/scheduler/intervalaction/edit-interval-action/edit-interval-action.component.ts (hunk 1 above)
Comment (badboy-huaqiao) [reply web/src/app/scheduler/intervalaction/edit-interval-action/edit-interval-action.component.ts]: thanks for your advice, it's done!

**Review by bnevis-i — APPROVED** [review]

**Pull request merged**